conf/supervisord-dev.conf

```diff
@@ -2,6 +2,22 @@
 nodaemon = true
 silent = true
 
+[program:init_db]
+command=python3 -m h.scripts.init_db --create --stamp
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal=KILL
+stopasgroup=true
+startsecs=0
+
+[program:init_elasticsearch]
+command=python3 -m h.scripts.init_elasticsearch conf/development.ini
+stdout_events_enabled=true
+stderr_events_enabled=true
+stopsignal=KILL
+stopasgroup=true
+startsecs=0
```

Comment:
Supervisor now automatically runs the new `init_db` and `init_elasticsearch` scripts when you run `make dev`. This is so you don't have to run `init_db` and `init_elasticsearch`  manually before running `make dev` when setting up a new dev environment.
  
Supervisor will run `init_db` and `init_elasticsearch` in parallel with booting up the Gunicorn worker, Celery worker, etc. This is faster than having `Makefile` or `tox.ini` run `init_db`, then `init_elasticsearch`, then `supervisor` in serial. There's no race condition here: if the other  programs finish booting up before `init_db` and `init_elasticsearch` have completed then any web requests or Celery tasks will be failing but once `init_db` and `init_elasticsearch` complete then stuff starts working without a restart.